website/pages/commands/operator/autopilot.mdx

```diff
@@ -97,3 +97,72 @@ Configuration updated!
 ```
 
 The return code will indicate success or failure.
+
+## state
+
+This command displays the current autopilot state.
+
+Usage: `consul operator autopilot state [options]`
+
+#### API Options
+
+@include 'http_api_options_client.mdx'
+
+@include 'http_api_options_server.mdx'
+
+#### Command Options
+
+- `-format` - Specifies the output format. Must be one of `[pretty|json]` and it defaults to `pretty`.
+
+#### Sample Output
+
+```
```

Comment:
Nit: I think it's really nice to show the command ran in the sample output if possible